allennlp/models/biattentive_classification_network.py

```diff
@@ -297,10 +297,12 @@ def decode(self, output_dict: Dict[str, torch.Tensor]) -> Dict[str, torch.Tensor
     def get_metrics(self, reset: bool = False) -> Dict[str, float]:
         return {metric_name: metric.get_metric(reset) for metric_name, metric in self.metrics.items()}
 
+    # The logic here requires a custom from_params.
```

Comment:
This is the `FeedForward` vs. `Maxout` logic, right?  I'd note that explicitly.  It'd be nice to find a way to fix it, but that can be another PR someday if someone wants to do it.